### PR TITLE
Remove deprecated roles

### PR DESCRIPTION
**Pull request by detiber**
Breaking out bugfix changes from https://github.com/openshift/openshift-ansible/pull/1773

### CODE REVIEW

**Comment by detiber** [comment]
@abutcher ptal

**Comment by detiber** [comment]
aos-ci-test

File: bin/cluster [hunk 1]
```diff
@@ -316,7 +316,7 @@ This wrapper is overriding the following ansible variables:
         meta_parser.add_argument('cluster_id', help='prefix for cluster VM names')
 
     meta_parser.add_argument('-t', '--deployment-type',
-                             choices=['origin', 'online', 'enterprise', 'atomic-enterprise', 'openshift-enterprise'],
```
Comment (abutcher) [inline bin/cluster]: @detiber Confirming we're gold to kill 3.0 in bin/cluster

Comment (detiber) [inline bin/cluster]: @abutcher yes, none of the provider playbooks would succeed with deployment_type=enterprise :)

Comment (abutcher) [inline bin/cluster]: Cool thought so

**Comment by abutcher** [comment]
lgtm

**Pull request merged**